lib/logitech_receiver/hidpp20.py

```diff
@@ -50,36 +51,63 @@
 	FEATURE_SET=0x0001,
 	FEATURE_INFO=0x0002,
 	DEVICE_FW_VERSION=0x0003,
+	DEVICE_UNIT_ID=0x0004,
 	DEVICE_NAME=0x0005,
 	DEVICE_GROUPS=0x0006,
+	DEVICE_FRIENDLY_NAME=0x0007,
+	KEEP_ALIVE=0x0008,
 	RESET=0x0020, # "Config Change"
-	DFUCONTROL=0x00C0,
-	DFUCONTROL_2=0x00C1,
+	CRYPTO_ID=0x0021,
+	TARGET_SOFTWARE=0x0030,
+	WIRELESS_SIGNAL_STRENGTH=0x0080,
+	DFUCONTROL_LEGACY=0x00C0,
+	DFUCONTROL_UNSIGNED=0x00C1,
+	DFUCONTROL_SIGNED=0x00C2,
```

Comment:
nitpick: Can you call them `DFU_CONTROL_*`?

Reply:
I gave them the names from the Logitech list. I am not sure what you want. Can you make the changes that you would like and then merge the pull request?

Reply:
Sure. I was saying to rename `DFUCONTROL_LEGACY` to `DFU_CONTROL_LEGACY`, etc. But I will do that myself an merge.

Reply:
In the list from https://github.com/pwr-Solaar/Solaar/issues/582 they are like  `DFUCONTROL_LEGACY` and not like `DFU_CONTROL_LEGACY`.

Reply:
I took that from #586, fixed now.